specification/metadata/schema/providerversion.md

```diff
@@ -0,0 +1,26 @@
+# Provider Version
+
+The ProviderVersion MAY be supplied to declare the version of logic by which the FOCUS dataset was generated and is separate from FOCUS Version. ProviderVersion allows for the provider to specify changes that may not result in a structural change in the data. It is suggested that the provider version use a versioning approach such as [SemVer](https://semver.org) version.
```

Comment:
@rileyjenk how can I understand the difference between provider version and Schema ID?

Reply:
@AWS-ZachErdman the examples should help explain this. SchemaID is a construct to identify the schema object, not a declaration of any versioning. Provider version supplies a mechanism by which the provider can specify version and communicate changes when they modify their logic/process to provide changes to generating the data.